frontend/src/component/project/Project/PaginatedProjectFeatureToggles/ProjectCleanupReminder/ProjectCleanupReminder.tsx

```diff
@@ -51,12 +49,22 @@ export const ProjectCleanupReminder: FC<{
     }
 
     const handleViewFlags = () => {
+        trackEvent('project-cleanup', {
+            props: {
+                eventType: 'view flags clicked',
+            },
+        });
         navigate(
             `/projects/${projectId}/features?${new URLSearchParams(query).toString()}`,
         );
     };
 
     const handleDismiss = () => {
+        trackEvent('project-cleanup', {
```

Comment:
do we have the same tracking for the old reminders at the flag level? if so are the names synced?